TempMail/ContentView.swift

```diff
@@ -8,49 +8,110 @@
 
 import SwiftUI
 
+
 struct ContentView: View {
+    @State var orientation: UIDeviceOrientation = UIDevice.current.orientation
 //    variable for Email struct
     @ObservedObject var email: Email = Email()
     //this variable is for later, when I implement a feature that will save user data even when the app is closed
     private let defaults = UserDefaults.standard
     
+    // TODO: can the header be abstracted? lots of repeated code...
```

Comment:
Maybe we can make another file with the VStack view, and then have it take in an email obj as a parameter from the ContentView. Since it is an Observable object the reference can be passed between the multiple views.
struct ContentView: View {
@ObservedObject var email: Email = Email()
var body: some View {
     HStack {
        if....
           VstackView(email)
        else 
            VstackView(email)
idk if this works tho but that's how I would envision it